src/localization/en.json

```diff
@@ -24,16 +24,25 @@
       "language": "Language:",
       "createdAt": "Created at:",
       "updatedAt": "Updated at:"
-    }
+    },
+    "statistics": "Statistics",
+    "expected": "Expected"
   },
 
   "editorContent": {
     "userNotSelected": "Please select a user first",
-    "totalWorkTime": "Total work time"
+    "workTime": "Work time"
   },
 
   "logged": "Logged:",
   "expected": "Expected:",
-  "total": "Total:"
+  "total": "Total:",
```

Comment:
Here there is no space after the colon, but in the finnish file there is. In the end either way is fine, but they should be consistent troughout the project. I personally have usually left the colons out from the localization, since if the colon is in localization variable, it cannot be left out easily if the word needs to be used without it, e.g. in title of some section, but if it isn't there, it can always be added easily with template literal in the component that uses it.